src/content/blog/whats-new-october-2023.mdx

```diff
@@ -0,0 +1,124 @@
+
+- [@lilnasy](https://github.com/lilnasy): $500 for outstanding contributions to the Astro codebase and community ⭐️
+- [@itsmatteomanf](https://github.com/itsmatteomanf): $250 for initiative taken translating our docs into Italian 🍾
+- [@alexanderniebuhr](https://github.com/alexanderniebuhr): $250 for leadership in the development of the Cloudflare adapter 🎉
+- [@HiDeoo](https://github.com/hideoo): $250 for taking ownership in Starlight's maintenance, support, and new features 🥳
+- [@OliverSpeir](https://github.com/oliverspeir): $250 for a significant presence and impact in Discord support 🎈
+
+### ViteConf
+
+This month, three Astro Core members presented talks at [ViteConf 2023](https://viteconf.org/23/).
+
+Learn more in our [ViteConf blog post](https://astro.build/blog/viteconf-2023/) and watch [the YouTube playlist](https://www.youtube.com/watch?v=veCxKeLl35A&list=PLqGQbXn_GDmkOsHI7-Wrbv1GgAA4tJZhg) of all the talks.
+
+### Content
+
+[Project: U2 Space Baby](https://www.chrsgrrtt.com/u2-space-baby): read about Chris Garrett's companion app for U2’s [residence at the Sphere](https://www.u2.com/splash) and see the [new Space Baby app for U2](https://u2spacebaby.u2.com/) live in action!
+
+[Code Fix: The SIBA Website's Move from Create-React-App to Astro](https://brittanisavery.com/post/move-siba-to-astro): read about converting a legacy CRA project to Astro by Brittani S. Avery
+
+[Translate routes in Astro with Content collection, subpage, optional trailingSlash and base support](https://www.webdesign-sopelnik.de/en/blog/translate-routes-for-astro-content-collections-or-subpages-with-trailingslash-and-base-support/): a guide to translating routes for content collections and subpages by Artur Sopelnik 
+
+## Showcase
+
+Each week on the Astro Community Call in Discord, we shine the spotlight on projects posted by community members in our `#showcase` channel from the following week. Here's a sample of what we saw this month!
+
+### Tools
+
+- [Darkmatter](https://getdarkmatter.dev/): A CMS-like interface for your Astro content collections (MacOS)
+- [Frontmatter CMS](https://frontmatter.codes/):  A Headless CMS right in your code editor - now supports Astro!
+- [Search Astro Documentation Raycast Extension](https://www.raycast.com/ElianCodes/search-astro-docs): search Astro docs directly within Raycast
+- [Set Up a Static Site With Astro on Kinsta](https://kinsta.com/docs/astro-static-site/): new guide for setting up a static site with Kinsta’s Application or Static Site hosting.
+
+### Themes
+
+- [Catonaut](https://github.com/AminoffZ/catonaut) by @aminoff
+- [Astro Supabase Starter](https://github.com/kevinzunigacuellar/astro-supabase) by @kevinzunigacuellar
+- [Engineering Blog](https://astro.build/themes/details/engineering-blog/) by @eddyvinck
+- [Ovidius](https://ovidius-astro-theme.netlify.app/) by @tomas_ba
```

Comment:
I think @Yan-Thomas’s comment got lost in the shuffle but I don’t see `@tomas_ba` associated with this theme?

[All the commits](https://github.com/JustGoodUI/ovidius-astro-theme/commits/main) are from `@astab` on GH.